### DISCUSSION TITLE
What new modifiers would be useful?

### DISCUSSION THREAD

**Discussion by tigert** [MobiFlight/MobiFlight-Connector#1549] [Ideas]
Sometimes I have been pondering the new cool modifiers and wanted to write down some ideas and maybe have a bit more discussion about what others would find useful.

- Timer / delay: delay an event for a given number of milliseconds
- Value ramp - kinda like the delay, but ramp a value from A to B over a given time. Fade a light, spool up a fan
 - Optionally have easing curve applied, so that the transition can appear more fluid/natural
- Format a number to a given number of decimals

**Comment by CaptainBobSim** [comment]
Format a number to a given number of decimals would be super helpful! It can be tricky to treat 1 as 1.000 for example
